### PR TITLE
[Merged by Bors] - Add brew to readme

### PR DESCRIPTION
**Pull request by eyalb181**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,6 +95,11 @@ Get the plugin [here](https://plugins.jetbrains.com/plugin/19772-mirrord).
 
 ### Installation
 
+You can use either
+```sh
+brew install metalbear-co/mirrord/mirrord
+```
```
Comment (aviramha) [inline README.md:101]: need to specify it's both Linux/macOS

Comment (eyalb181) [reply README.md:101]: Why? We don't do it on the website

Comment (aviramha) [reply README.md:101]: valid point

**Review by aviramha — APPROVED** [review]
fix lint - LGTM

**Pull request closed without merging**